src/components/Footer.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="footer__container d-flex flex-row justify-content-end mx-5 mt-3">
+    <div class="footer-items__container d-flex flex-row my-auto mx-3">
+      <div class="footer__item d-flex flex-column align-items-center justify-items-center">
```

Comment:
Make a tooltip for it
also link it to https://t.me/dnjco

Reply:
Used Bootstrap tooltip